samples/HealthChecksSample/DbConnectionHealthCheck.cs

```diff
@@ -51,11 +51,11 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
                 }
                 catch (DbException ex)
                 {
-                    return HealthCheckResult.Failed(exception: ex);
+                    return new HealthCheckResult(status: context.Registration.FailureStatus, exception: ex);
```

Comment:
@unaizorrilla - I would expect your code to look a lot like this. This pattern allows the app developer to configure the status.

Reply:
Hi @rynowak ,
 
Yes, when the new bits will be available on MyGet I perfom the changes on HealthChecks